src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -97,13 +97,8 @@ Json::Value doAccountLines (RPC::Context& context)
     std::string strIdent (params[jss::account].asString ());
     AccountID accountID;
 
-    if (auto jv = RPC::accountFromString (accountID, strIdent))
-    {
-        for (auto it = jv.begin (); it != jv.end (); ++it)
-            result[it.memberName ()] = it.key ();
-
+    if ((result = RPC::accountFromString (accountID, strIdent)))
```

Comment:
@vinniefalco This is a different case.  It's an assignment to a pre-existing variable in an `if`.  I think the warning serves a useful purpose.  And the extra parentheses silence the warning.